src/apps/testapps/testH3UniEdge.c

```diff
@@ -124,13 +124,31 @@ SUITE(h3UniEdge) {
     }
 
     TEST(getH3UnidirectionalEdgeFromPentagon) {
+        H3Index pentagons[NUM_PENTAGONS] = {0};
```

Comment:
nit: `= {H3_NULL}` now? I prefer using the constant instead of 0 when comparing H3Index to be clear what the expected type is.

Reply:
Agree for comparison, and updated below, but this is actually initializing the array to `0`, right? According to [this SO answer](https://stackoverflow.com/a/201116/380487), this wouldn't work for any non-`0` value, so a) it's idiomatic in a way that `{CONST_RESOLVING_TO_ZERO}` isn't, and b) putting in `{H3_NULL}` requires the reader to know that `H3_NULL` resolves to zero in order to understand that we're initializing the whole array, not just the first element.